modules/Activities/report_activitySpread_formGroup.php

```diff
@@ -97,7 +97,7 @@
         ->sortable(['surname', 'preferredName'])
         ->format(function ($student) use ($guid) {
             $name = Format::name('', $student['preferredName'], $student['surname'], 'Student', true);
-            return Format::link($_SESSION[$guid]['absoluteURL'].'/index.php?q=/modules/Students/student_view_details.php&gibbonPersonID='.$student['gibbonPersonID'].'&subpage=Activities', $name);
+            return Format::link($session->get('absoluteURL').'/index.php?q=/modules/Students/student_view_details.php&gibbonPersonID='.$student['gibbonPersonID'].'&subpage=Activities', $name);
```

Comment:
I suspect that this would throw an error. Check the syntax but also consider the lines above and below this line: what do you notice?